projects/swimlane/ngx-ui/src/lib/components/select/select-option.directive.ts

```diff
@@ -13,6 +14,17 @@ export class SelectOptionDirective implements SelectDropdownOption {
   @Input() name = '';
   @Input() value: any;
 
+  @Input() tooltipAlignment: AlignmentTypes = AlignmentTypes.center;
```

Comment:
Are all of these inputs needed? The tooltip directive should be providing them.

Reply:
It does provide them but it does not work on `ngx-select-option`. Only works on the parent `ngx-select` and we need a way to display tooltips with custom templates for dropdown options.